# config to find camera position in world

world_frame: "world"
calibration_marker_frame: "arena_tag"
camera_frame: "arena_camera_optical"

position_marker_in_world:
  x: 0.0    # World origin x
  y: 0.0    # World origin y  
  z: 0.0    # World origin z (typically ground level)
orientation_in_world:
  # needed because april tag coordinate convention differs from ros coordinate convention
  roll: 0.0    # Rotation around X axis (radians)
  pitch: 3.14159   # Rotation around Y axis (radians)
  yaw: 1.5708     # Rotation around Z axis (radians)